packaging/windows/clisdk/generatenupkg.ps1

```diff
@@ -40,8 +40,8 @@ function GenerateNupkg
     $SdkBundlePath = [System.IO.Path]::GetFullPath($SdkBundlePath)
     $NuspecFileName = [System.IO.Path]::GetFileName($NuspecFile)
     $TempNuspecFile = [System.IO.Path]::Combine($OutputDirectory, $NuspecFileName)
-    (Get-Content $NuspecFile) -replace '\[DOTNET_BUNDLE\]', $SdkBundlePath | Set-Content $TempNuspecFile
-    & $NuGetExe pack $TempNuspecFile -Version $NugetVersion -OutputDirectory $OutputDirectory
```

Comment:
What was wrong with -Version here, exactly?

Reply:
Also, it occurs to me that we should be using -Properties and the $foo$ syntax would be preferable for anything other than Version. The search and replace is clunky (though we were doing it already for DOTNET_BUNDLE.

Reply:
The NuSpec file in the packages output directory retains the dummy version if we use "-Version"; if we modify the version in the NuSpec, it is directly reuseable from the output directory.

Reply:
>but more importantly the NuSpec is embedded in the nupkg - I feel we want the version to be consistent in the package.

 The nuspec embedded in the package will be impacted by -Version and -Properties